rust/BUILD.md

```diff
@@ -0,0 +1,42 @@
+# Building and testing the Rust bindings
+
+These instructions require cargo and rustc.
+To get these follow the instructions at [https://rustup.rs](https://rustup.rs)
+The instructions compile the onnxruntime along with the bindings,
+so require `cmake`, a python 3 interpreter, clang (needed to parse the C headers to generate the Rust bindings),
```

Comment:
It can remove dependency of clang as auto generating bindings code on the GitHub Actions.
[like this](https://github.com/VOICEVOX/onnxruntime-rs/blob/master/.github/actions/auto_gen_bind_pr/action.yaml)

Reply:
Yeah, we would have to check-in auto-generated code then. I think it will be less work to auto-generate the bindings with bindgen in each build.

Reply:
OK I understand.